docs/api-v1.md

```diff
@@ -10,9 +10,11 @@
     "username": "jake",
     "lastname": "Smith",
     "firstname": "Jacob",
+    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1NDM2ODUwMzYsImV4cCI6MTU0MzY5MjIzNiwianRpIjoiczlKSmErMGpPb2FGaWFxMTd6VHU5dz09IiwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdCIsInN1YiI6InNpbWkuZ3VsaSJ9.khH8KxrUWBco5eoiMHVgVRk2BMx1cMpvjDJH8j24sXs",
```

Comment:
I do not think that the token should be here. Normally the token should only be returned when `/login` is called. So I suggest to change what is returned from the login route from a user object to just the token or the user object + token sepreatly. Therefore when retrieving a user (e.g. `/user`) the token is not exposed.

```json
{
    "token": "xxx",
    "user": {
        "id": 1,
        "username": "jake",
        "lastname": "Smith",
        "firstname": "Jacob",
        "email": "jake@example.com"
    }
}
```

```suggestion
```

Reply:
If we update this, we also have to adjust the login method in LoginController class

Reply:
Ok, I updated it.